### ISSUE TITLE
Portable toilet address

### ISSUE THREAD

**Issue by HellMapGoesCoding** [Davis-Klavins/osm-latvia-bot#4]
Hi! The bot [added an address](https://www.openstreetmap.org/node/9792362155/history) to a `amenity=toilets`; `portable=yes` node. This doesn't seem correct. I guess I see how a `amenity=toilets` building (or approximate node) would merit an address, but I think this should exclude `portable=yes`. The address also was added/updated on a [proper building](https://www.openstreetmap.org/way/243748993/history) within the parcel, so I'm not exactly sure what happened - is the bot allowed to add the same address multiple times?

**Comment by Davis-Klavins** [comment]
Addresses to amenity=toilets are added only to ways since [August 12](https://github.com/Davis-Klavins/osm-latvia-bot/commit/1b4906a7be4c51fcc77517fb70e986ba3cc17c55). When reverting deleted feature, it was reverted as it was before deletion - with an address (back then addresses to amenity=toilets were added also to nodes). Bot would have corrected the feature (deleted address) when being run next time.
In short, addresses are added to buildings (or as address nodes) and all objects listed in [tags_4_addresses.csv](https://github.com/Davis-Klavins/osm-latvia-bot/blob/main/tags_4_addresses.csv). Extended description is planned to be added in the [Wiki](https://wiki.openstreetmap.org/wiki/Automated_edits/Latvia-bot).

**Comment by HellMapGoesCoding** [comment]
I see now. Thanks for clarifying.
